discovery-infra/test_infra/config/_cluster_config.py

```diff
@@ -0,0 +1,55 @@
+from dataclasses import dataclass
+
+from test_infra.config import EnvConfig
+from test_infra.helper_classes.cluster_config import ClusterConfig, ClusterInstallConfig
+
+
+@dataclass
+class DefaultInstallConfig(ClusterInstallConfig):
+
+    def __post_init__(self):
+        self.iso_download_path = self.iso_download_path \
+            if self.iso_download_path is not None else EnvConfig.get("iso_download_path")
+        self.iso_image_type = self.iso_image_type \
+            if self.iso_image_type is not None else EnvConfig.get("iso_image_type")
+        self.nodes_count = self.nodes_count \
+            if self.nodes_count is not None else EnvConfig.get("num_nodes")
+        self.vip_dhcp_allocation = self.vip_dhcp_allocation \
+            if self.vip_dhcp_allocation is not None else EnvConfig.get("vip_dhcp_allocation")
+        self.download_image = self.download_image \
+            if self.download_image is not None else EnvConfig.get("download_image")
+        self.platform = self.platform if self.platform is not None else EnvConfig.get("platform")
+        self.is_ipv6 = self.is_ipv6 if self.is_ipv6 is not None else EnvConfig.get("is_ipv6")
+        self.service_network_cidr = self.service_network_cidr \
+            if self.service_network_cidr is not None else EnvConfig.get("service_cidr")
+        self.cluster_network_cidr = self.cluster_network_cidr \
+            if self.cluster_network_cidr is not None else EnvConfig.get("cluster_cidr")
+        self.cluster_network_host_prefix = self.cluster_network_host_prefix \
+            if self.cluster_network_host_prefix is not None else EnvConfig.get("host_prefix")
+
+
+@dataclass
+class DefaultClusterConfig(ClusterConfig):
+
+    def __post_init__(self):
+        """
+        If not set get from environment variables
```

Comment:
weird sentence
maybe ``Apply default configuration values from environment variables``
and do it a oneline docstring

Reply:
Forgot to change it :)